cloud/kubernetes/helm/values.yaml

```diff
@@ -9,7 +9,7 @@
 # Declare variables to be passed into your templates.
 
 #Required parameter to be passed for all cloud providers
-cpType: minikube      #cpType can be minikube, aws, awsfargate, azure, openshift OR gcp
+cpType: minikube      #cpType can be minikube, aws, awsfargate, awsefs, azure, openshift OR gcp
```

Comment:
I see a new cloud provider type (cpType) "awsefs" added here, do you mean - 
Amazon Elastic File System?
What is the significance of it?
What is the difference between aws & aswefs?

Reply:
This flag is added when user is trying to deploy BE app on aws with efs storageclass, as it requires efs csi driver chart to be installed in cluster